classes/XLite/Module/PureClarity/Personalization/Core/Task/FeedRunner.php

```diff
@@ -132,14 +135,34 @@ protected function sendFeeds(array $feedTypes)
         }
 
         if (in_array(Feed::FEED_TYPE_USER, $feedTypes)) {
+            $feedTypes = $this->removeFeedFromState($feedTypes, Feed::FEED_TYPE_USER);
```

Comment:
You've placed the others at the bottom of each block, any difference here?

Reply:
C&P error, should be at the bottom, fixed now :)